flink/v1.13/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergFilesCommitter.java

```diff
@@ -166,6 +167,29 @@ public void testMaxContinuousEmptyCommits() throws Exception {
     }
   }
 
+  @Test
+  public void testNoneEmptyCommits() throws Exception {
+    table.updateProperties().set(MAX_CONTINUOUS_EMPTY_COMMITS, "0").commit();
+
+    JobID jobId = new JobID();
+    long checkpointId = 0;
+    long timestamp = 0;
+    try (OneInputStreamOperatorTestHarness<WriteResult, Void> harness = createStreamSink(jobId)) {
+      harness.setup();
+      harness.open();
+
+      assertSnapshotSize(0);
+
+      for (int i = 1; i <= 9; i++) {
+        harness.processElement(new StreamRecord<>(WriteResult.builder().build()));
+        harness.snapshot(++checkpointId, ++timestamp);
```

Comment:
In Iceberg, we don't use the return value of the `++` operator because it is confusing for readers.